app/scripts/modules/core/src/serverGroup/serverGroupWriter.service.ts

```diff
@@ -124,13 +124,14 @@ export class ServerGroupWriter {
     params.region = serverGroup.region;
     params.credentials = serverGroup.account;
     params.cloudProvider = serverGroup.type || serverGroup.provider;
+    const newSize: string = Object.keys(params.capacity)
+      .map(k => `${k}: ${(params.capacity as any)[k]}`)
```

Comment:
This might read a bit nicer
```suggestion
      .map((k: keyof ICapacity) => `${k}: ${params.capacity[k]}`)
```

Reply:
idk 🤷‍♂️

Reply:
yeah, that is better